### PR TITLE
PP-8239 Insert into history table for gateway_account_credentials

### PR DESCRIPTION
**Pull request by stephencdaly**
Use the `@Customizer` annotation on GatewayAccountCredentialsEntity in the same way as used for RefundEntity so that JPA automatically inserts into the gateway_account_credentials_history table when the entity is updated.

### CODE REVIEW

File: src/main/resources/migrations.xml [hunk 1]
```diff
@@ -1661,4 +1661,22 @@
         <addNotNullConstraint tableName="gateway_accounts" columnName="send_payer_email_to_gateway"/>
     </changeSet>
 
+    <changeSet id="build gateway_account_credentials_history table" author="">
```
Comment (kbottla) [inline src/main/resources/migrations.xml]: Can we do DB migration in separate PR ?

Comment (stephencdaly) [reply src/main/resources/migrations.xml]: oh yeh good point

Comment (stephencdaly) [reply src/main/resources/migrations.xml]: migration was moved to another PR and run on all environments

**Review by kbottla — APPROVED** [review]
LGTM!
